display/ui_settings.py

```diff
@@ -1,26 +1,28 @@
+from typing import Tuple, Optional
+
 from PyQt5.QtCore import *
 from PyQt5.QtGui import *
 
 
 class TextSetting:
-    def __init__(self, fontName, fontSize, bold):
+    def __init__(self, fontName: str, fontSize: int, bold: bool):
         if bold:
             self.font = QFont(fontName, fontSize, QFont.Bold)
-        else:
```

Comment:
Why remove this else?

Reply:
The `__init__` only had this as control logic, so it just seemed like a pointless `else` since it was only change one thing if the `bold` bool was `true`.

Reply:
I think this code will overwrite the font to be non-bold even if `bold` is `True`. That was the reason for including the `else` initially.

Reply:
I think this code will overwrite the font to be non-bold even if `bold` is `True`. That was the reason for including the `else`initially.